app/models/user.rb

```diff
@@ -16,7 +16,13 @@ class Signin
 
     validates_format_of :email, with: /\A([^\s]+)((?:[-a-z0-9]\.)[a-z]{2,})\z/i, unless: Proc.new { |o| o.email.blank? }
     validates :email, :password, presence: true
- end
+  end
+
+  def self.signin(args)
+    Signin.new(args).tap do |signin|
+      signin.errors.add(:base, :invalid, message: 'Enter a valid email and password') if signin.valid?
```

Comment:
Can we use I18n.t() here?

Reply:
The translation is actually performed based on the keyword `:invalid`, the message value is more for the console.
This is the line that does will be used https://github.com/ministryofjustice/tax-tribunals-datacapture/pull/739/files#diff-44438ce218f5287c58d0017f965d888715635d94280669896f75841fbd7b4cd7R704